### PR TITLE
Cleanup misc section

### PR DESCRIPTION
**Pull request by KhushrajRathod**
Most of the stuff was in the wrong category

### CODE REVIEW

File: app/src/main/res/xml/pref_misc.xml [hunk 1]
```diff
@@ -1,91 +1,92 @@
...
             android:title="Force headset switch (Huawei default)" />
-    </PreferenceCategory>
-    <EditTextPreference
-        android:defaultValue="-1"
-        android:key="key_misc_rounded_corners"
-	android:title="Set rounded corners diameter"
-	android:inputType="number" />
-    <EditTextPreference
+	</PreferenceCategory>
+	<PreferenceCategory android:title="Backlight">
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:24]: Well rounded corners isn't about backlight.
Maybe change it to screen?

File: app/src/main/res/xml/pref_misc.xml [hunk 2]
```diff
@@ -1,91 +1,92 @@
...
         android:title="Bluetooth workarounds" />
-    <Preference android:title="Securize"
+		<Preference android:title="Securize"
         android:key="key_misc_securize" />
-    <CheckBoxPreference
+		<CheckBoxPreference
         android:defaultValue="false"
         android:key="key_misc_remotectl"
         android:title="Allow takeover of the device by phh for debugging" />
+		<ListPreference
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:57]: Can be moved to display

File: app/src/main/res/xml/pref_misc.xml [hunk 3]
```diff
@@ -1,91 +1,92 @@
...
+    	    android:defaultValue="1"
+    	    android:entries="@array/pref_misc_fps_divisor"
+    	    android:entryValues="@array/pref_misc_fps_divisor_values"
+    	    android:key="key_misc_fps_divisor"
+    	    android:title="FPS Divisor" />
+		<ListPreference
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:63]: Can be moved to display

File: app/src/main/res/xml/pref_misc.xml [hunk 4]
```diff
@@ -1,91 +1,92 @@
...
+    	    android:defaultValue="-1"
+    	    android:entries="@array/pref_misc_display_fps"
+    	    android:entryValues="@array/pref_misc_display_fps_values"
+    	    android:key="key_misc_display_fps"
+    	    android:title="Force FPS" />
+		<ListPreference
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:69]: Maybe move it to display?

File: app/src/main/res/xml/pref_misc.xml [hunk 5]
```diff
@@ -1,91 +1,92 @@
...
+            android:title="Expose Aux cameras (Qualcomm)" />
+		<CheckBoxPreference
+    		android:defaultValue="false"
+    		android:key="key_misc_force_camera2api_hal3"
+    		android:title="Force Enable Camera2API HAL3" />
+		<CheckBoxPreference
+            android:defaultValue="false"
+            android:key="key_misc_force_navbar_off"
+            android:title="Force navigation bar disabled" />
+		<CheckBoxPreference
```
Comment (phhusson) [inline app/src/main/res/xml/pref_misc.xml:87]: Could be moved with key_huawei_headset_fix as "Audio" section

File: app/src/main/res/xml/pref_misc.xml (hunk 1 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:24]: Ya wait I don't know why I put that there

File: app/src/main/res/xml/pref_misc.xml (hunk 2 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:57]: How is remote control in display?

File: app/src/main/res/xml/pref_misc.xml (hunk 3 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:63]: Ok

File: app/src/main/res/xml/pref_misc.xml (hunk 4 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:69]: Ok

File: app/src/main/res/xml/pref_misc.xml (hunk 5 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:87]: How is navbar in audio?

File: app/src/main/res/xml/pref_misc.xml (hunk 2 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:57]: Ah right I see now

File: app/src/main/res/xml/pref_misc.xml (hunk 5 above)
Comment (KhushrajRathod) [reply app/src/main/res/xml/pref_misc.xml:87]: Right ofc, Done

**Comment by KhushrajRathod** [comment]
Anything else to change?

**Pull request merged**